common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -0,0 +1,387 @@
+// unknown === can't/haven't checked
+// iiif | pdf | video | audio === checked manifest and can render
+// none === checked manifest and know we can't render it
+type MultiVolumePreviewProps = {|
+  children: Node,
+|};
+
+const MultiVolumePreview = ({ children }: MultiVolumePreviewProps) => {
+  const multiPreview = useRef();
+  const isOnScreen = useOnScreen({ ref: multiPreview, threshold: 0.75 });
+
+  return (
+    <MultiVolumeContainer ref={multiPreview} isOnScreen={isOnScreen}>
+      {children}
+    </MultiVolumeContainer>
+  );
+};
+
+const IIIFPresentationPreview = ({
+  itemUrl,
+  childManifestsCount = 0,
+}: Props) => {
+  const [viewType, setViewType] = useState<ViewType>('unknown');
+  const [imageThumbnails, setImageThumbnails] = useState([]);
+  const [hasThumbnails, setHasThumbnails] = useState(false);
+  const [imageTotal, setImageTotal] = useState(0);
+  const iiifPresentationManifest = useContext(ManifestContext);
+
+  useEffect(() => {
+    const allImages = imageThumbnails.reduce(
+      (acc, thumb) => [...acc, ...thumb.images],
+      []
+    );
+    setHasThumbnails(allImages.length > 0);
+  }, [imageThumbnails]);
+
+  useEffect(() => {
+    if (iiifPresentationManifest) {
+      setViewType(getManifestViewType(iiifPresentationManifest));
+      setImageTotal(getCanvases(iiifPresentationManifest).length);
+      setImageThumbnails(
+        previewThumbnails(
+          iiifPresentationManifest,
+          orderedStructuredImages(structuredImages(iiifPresentationManifest)),
+          5
+        )
+      );
+    }
+  }, [iiifPresentationManifest]);
+
+  if (viewType === 'unknown' || viewType === 'pdf') {
+    return (
+      <div className="container">
+        <div className="grid">
+          <div className={grid({ s: 12, m: 12, l: 12, xl: 12 })}>
+            <Space
+              v={{
+                size: 'm',
+                properties: ['margin-top', 'margin-bottom'],
+              }}
+            >
+              <Button
+                type="primary"
+                url={`/works/${itemUrl.href.query.workId}/items`}
+                trackingEvent={{
+                  category: 'ViewBookNonJSButton',
+                  action: 'follow link',
+                  label: itemUrl.href.query.workId,
+                }}
+                text="View the item"
+                link={itemUrl}
+              />
+            </Space>
+          </div>
+        </div>
+      </div>
+    );
+  }
+
+  if (viewType === 'iiif' && hasThumbnails) {
+    return (
+      <WobblyRow>
+        <PresentationPreview>
+          <NextLink {...itemUrl}>
+            <a
+              className="plain-link"
+              onClick={() => {
+                trackEvent({
+                  category: 'IIIFPresentationPreview',
+                  action: 'follow link',
+                  label: itemUrl.href.query.workId,
+                });
+              }}
+            >
+              {childManifestsCount === 0 &&
```

Comment:
Lost as to how this makes sense?
If we have no childManifests, which are sent from a parent, use `imageThumbnails`?

Sure it makes sense, but it's not massively obvious.